### ISSUE TITLE
2 Shippable configs in the same repo

### ISSUE THREAD

**Issue by alexvasiu** [Shippable/support#4621]
We have in my repo something like this:
* backend/ (Python)
* frontend/ (NodeJs)

How to add one config for each directory? (one for python and one for NodeJs)

**Comment by trriplejay** [comment]
Shippable only supports a single shippable.yml file in the root of your repo for running standard CI.

However, we do provide a utility that allows you to determine which files have changed, and you can use that information to decide which script to run, or which yml commands to execute.

When you run a build on Shippable, you'll have access to the `shipctl` utility, and we've added a function called `get_git_changes` that can tell you information about what has changed in your repository.  Documentation for that is here: http://docs.shippable.com/platform/tutorial/workflow/using-shipctl/#get_git_changes

so basically, if all of your python code is in a folder called 'backend', and all of your nodejs code is in a folder called 'frontend', you should be able to add something like this to your yml:

```yml
build:
  ci:
    - changed_dirs=$(shipctl get_git_changes --path="$SHIPPABLE_BUILD_DIR" --directories-only --depth=1)
    - if [[ " ${changed_dirs[@]} " =~ " backend " ]]; then ./run_backend_tests.sh; fi
    - if [[ " ${changed_dirs[@]} " =~ " frontend " ]]; then ./run_frontend_tests.sh; fi

```

you might need to tweak it a bit for your scenario, but let me know if you have any questions.

**Comment by alexvasiu** [comment]
@trriplejay Thank you 👍
